src/components/AddTicket.css

```
@@ -0,0 +1,54 @@
+.createTicketForm {
+  background-color: #163737; 
+  display: block;
+  padding: 1rem;
+  max-width: 350px;
+  border-radius: 0.5rem;
+  box-shadow: 0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05);
+  margin: 0 auto; 
+}
+
+.ticketTitle {
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  font-weight: 600;
+  text-align: center;
+  color: #fff; 
+}
+
+.createTicket-input-container {
+  position: relative;
+}
+
+.createTicket-input-container input, .createTicketForm button {
+  outline: none;
+  border: 1px solid #e5e7eb;
+  margin: 8px 0;
+}
+
+.createTicket-input-container input {
```

```suggestion
.createTicket-input-container input, .createTicket-input-container textarea {
```
Adding support for textarea